src/hooks/partners/useVerifyPartner.ts

```diff
@@ -0,0 +1,19 @@
+import { verifyPartner } from '@/services/partners/verifyPartner'
+import { useMutation } from '@tanstack/react-query'
+import { useEffect } from 'react'
+
+export function useVerifyPartner(partner?: string): boolean {
+  const { data, mutate } = useMutation({
```

Comment:
It is not a mutation... It is a GET request

Reply:
arf bien vu je modifie